src/Handler/Table/Create/CreateTableHandler.php

```diff
@@ -42,7 +42,7 @@ public function __invoke(
 
         // validate
         assert($command->getPath()->count() === 1, 'CreateTableCommand.path is required and size must equal 1');
-        assert(!empty($command->getTableName()), 'CreateTableCommand.tableName is required');
```

Comment:
fakt nekdo pojmenuje tabulku `0`? 😱 🤦 Ale OK.

Reply:
je to tak aj sloupce